projects/storefrontlib/src/cms-components/order-confirmation/replenishment-order-confirmation.module.ts

```diff
@@ -0,0 +1,56 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { ReactiveFormsModule } from '@angular/forms';
+import {
+  CmsConfig,
+  FeaturesConfigModule,
+  I18nModule,
+  provideDefaultConfig,
+} from '@spartacus/core';
+import { CardModule } from '../../shared/components/card/card.module';
+import { FormErrorsModule } from '../../shared/index';
+import { CartSharedModule } from '../cart/cart-shared/cart-shared.module';
+import { PromotionsModule } from '../checkout/components/promotions/promotions.module';
+import { PwaModule } from './../../cms-structure/pwa/pwa.module';
+import { OrderConfirmationItemsComponent } from './components/order-confirmation-items/order-confirmation-items.component';
+import { OrderConfirmationOverviewComponent } from './components/order-confirmation-overview/order-confirmation-overview.component';
+import { OrderConfirmationThankYouMessageComponent } from './components/order-confirmation-thank-you-message/order-confirmation-thank-you-message.component';
+import { OrderConfirmationTotalsComponent } from './components/order-confirmation-totals/order-confirmation-totals.component';
+import { OrderConfirmationGuard } from './guards/order-confirmation.guard';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    CartSharedModule,
+    CardModule,
+    PwaModule,
+    PromotionsModule,
+    I18nModule,
+    ReactiveFormsModule,
+    FeaturesConfigModule,
+    FormErrorsModule,
+  ],
+  providers: [
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        ReplenishmentConfirmationMessageComponent: {
```

Comment:
Why do you create new CMS components if you reuse the order ones in Spartacus?

Reply:
To stay consistent to what we were doing and I guess it makes sense because you want replenishment and not the normal order confirmation/and others (even if it's a dummy component). More specifically, how it was done for approval!